frontend/src/scene/sign-in/SignIn.js

```diff
@@ -47,18 +49,23 @@ const SizedLogo = styled(Logo)`
 class SignIn extends React.Component {
     render() {
         const location = this.props.location.pathname;
+        // this view is forbidden for authenticated user
+        if (this.props.userStore.isAuthenticated) {
+            console.log('Hello');
```

Comment:
This kind of logging should not be left around.

Reply:
removed